### DISCUSSION TITLE
Best way to watch for changes

### DISCUSSION THREAD

**Discussion by IanVS** [PaloAltoNetworks/docusaurus-openapi-docs#412] [Q&A] [answered]
Currently, I have an npm script that cleans the openapi docs and then re-generates them, which I run after making any changes to my schema.  But then the running docusaurus server throws errors, and I have to restart it.  Is there a better workflow for making changes and seeing them on the page?

**Comment by sserrata** [comment, marked as answer]
Currently, clean and re-generate (while server is stopped) is the best way to reflect changes to the OpenAPI spec.

There's a different approach whereby we'd generate the MDX in the `.docusaurus` directory (which allowed for hot reloading)  but we chose the current direction to maintain better compatibility with non-generated/API docs and `plugin-content-docs`.
